.github/workflows/release.yml

```
@@ -0,0 +1,153 @@
+        run: bash -n install.sh && bash -n scripts/build-exe.sh && bash -n scripts/publish-release.sh
+
+  docker-build:
+    name: Docker image builds + healthcheck
+    runs-on: ubuntu-latest
+    timeout-minutes: 12
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build image
+        run: docker build -t career-ops:ci .
+      - name: Boot + probe /api/health
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:2:1: i appears unused. Verify use (or export if used externally)

(shellcheck)